drivers/modem/Kconfig.quectel-bg9x

```diff
@@ -0,0 +1,68 @@
+# Quectel BG9x driver options
+
+# Copyright (c) 2020 Analog Life LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config MODEM_QUECTEL_BG9X
+	bool "Enable quectel modem driver"
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	select MODEM_SOCKET
+	select NET_OFFLOAD
+	select NET_SOCKETS_OFFLOAD
+	imply GPIO
+	help
+	  Choose this setting to enable quectel BG9x LTE-CatM1/NB-IoT modem
+	  driver.
+
+config MODEM_QUECTEL_BG9X_NAME
```

Comment:
Is there a reason this needs to be a Kconfig?

Reply:
Yeah, doesn't seem to be necessary - Was a copy-paste from the SARA-R4 driver. Will remove..